nginx/nginx.conf

```diff
@@ -0,0 +1,27 @@
+# For opencti
+server {
+    listen 80;
+    server_name opencti.investigate.africa;
+
+    location / {
+        proxy_pass http://opencti:8080;
+        proxy_set_header Host $host;
+        proxy_set_header X-Real-IP $remote_addr;
+        proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+        proxy_set_header X-Forwarded-Proto $scheme;
+    }
+}
+
+# For portainer
+server {
+    listen 80;
+    server_name portainer.investigate.africa;
```

Comment:
Will these `server_name` work even when I run this locally?

Reply:
> Will these `server_name` work even when I run this locally?

No, the reverse DNS for the server_names above point to the `deployment host IP`, to use nginx locally we'd need to have that point to `localhost`. Do we really need to use nginx locally?

Reply:
Cool. Can the `nginx.conf` read/be set from ENV vars? That way we wouldn't need to change the code for every domain change.

Reply:
Since we're using docker, use of env vars and templates is supported natively since [`v1.19`](https://github.com/docker-library/docs/tree/master/nginx#using-environment-variables-in-nginx-configuration-new-in-119)

Reply:
Is this implemented?